### PR TITLE
Remove service tests from flaky lists after recent revamp

### PR DESCRIPTION
**Pull request by thockin**

### CODE REVIEW

**Comment by thockin** [comment]
@k8s-bot test this please

kicking to see if I can make it flake

**Comment by thockin** [comment]
@k8s-bot test this please

kicking to see if I can make it flake

**Comment by thockin** [comment]
@k8s-bot test this please

kicking to see if I can make it flake

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
Quinton says this will trigger 15 more runs :)

**Comment by thockin** [comment]
Nope, it only triggered 2 runs.

@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

File: hack/jenkins/e2e.sh [hunk 1]
```diff
@@ -180,12 +176,6 @@ GCE_PARALLEL_FLAKY_TESTS=(
     "Namespaces.*should\sdelete\sfast"
     "PD"
     "ServiceAccounts"
-    "Services.*change\sthe\stype"
```
Comment (ixdy) [inline hack/jenkins/e2e.sh]: this one flaked this morning: https://goto.google.com/k8s-test/view/Flaky/job/kubernetes-e2e-gce-parallel-flaky/7453/

File: hack/jenkins/e2e.sh [hunk 2]
```diff
@@ -180,12 +176,6 @@ GCE_PARALLEL_FLAKY_TESTS=(
...
-    "Services.*functioning\sexternal\sload\sbalancer"
-    "Services.*identically\snamed"
-    "Services.*release.*load\sbalancer"
-    "Services.*endpoint"
```
Comment (ixdy) [inline hack/jenkins/e2e.sh]: `Services should serve multiport endpoints from pods` flaked in https://goto.google.com/k8s-test/view/Flaky/job/kubernetes-e2e-gce-parallel-flaky/7477/ (though so did lots of tests...)

Comment (ixdy) [inline hack/jenkins/e2e.sh]: also in 7324 (Oct 15) and some earlier builds.

Comment (thockin) [inline hack/jenkins/e2e.sh]: 7324 was a "All nodes should be ready" error and a 10 minute timeout (must be something GCE-related).

7477 were a bunch of related timeouts.

I still assert this test does not belong in flaky

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by ixdy** [comment]
sorry, failure caused by restarting Jenkins.
@k8s-bot test this please

**Comment by ixdy** [comment]
Jenkins is failing for all the wrong reasons.
@k8s-bot test this

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin-cc** [comment]
A lot of the remaining flakes are just waiting for load-balancers to come
up.  We can make those timeouts longer, but it appears that sometimes it
just doesn't work.

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
so far every failure has been a DIFFERENT flake - not this test.

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
I'd like to propose merging this now.  Any flakes that have happened  have been GCE related

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by thockin** [comment]
@k8s-bot test this please

File: hack/jenkins/e2e.sh [hunk 3]
```diff
@@ -180,12 +176,6 @@ GCE_PARALLEL_FLAKY_TESTS=(
...
-    "Services.*up\sand\sdown"
```
Comment (ixdy) [inline hack/jenkins/e2e.sh]: https://goto.google.com/k8s-test/job/kubernetes-e2e-gce-parallel-flaky/7619/ shows a failure of this test.

There were also PD test failures on the same build; not sure if related.

**Comment by ixdy** [comment]
It seems like the service and PD tests are less stable on the parallel build, and they often fail together when they flake. It's not immediately clear to me whether the PD tests are causing more instability, or if there's just general instability in our parallel tests.

**Comment by ixdy** [comment]
The one remaining service test on the non-parallel flaky build seems fine, though.

**Comment by thockin** [comment]
This flake is just a case of waiting for GCE.  It's happening less and
less, so I'd still like to get these marked un-flaky so they can run on PRs.

**Comment by thockin** [comment]
@k8s-bot test this please

**Comment by ixdy** [comment]
ok, lgtm.

**Pull request merged**